packages/wasm-dpp/lib/test/fixtures/getRawChainAssetLockProofFixture.js

```diff
@@ -0,0 +1,21 @@
+const { default: loadWasmDpp } = require('../../../dist');
```

Comment:
For the fixtures, I would suggest that if you don't need to make any changes, keep using the old ones. Thay way if something gets updated in the js-dpp it would be instantly reflected in the tests